src/components/training/UpcomingTrainingSection.js

```diff
@@ -31,13 +28,14 @@ const UpcomingTrainings = ({ curriculum }) =>
       </React.Fragment>
     )
   })
+}
 
-const UpcomingTrainingSection = ({ curriculum }) => (
+const UpcomingTrainingSection = ({ curriculum, type }) => (
   <React.Fragment>
     {curriculum ? (
       <React.Fragment>
-        <H3 style={{ marginTop: '1em' }}>Upcoming courses</H3>
-        <UpcomingTrainings curriculum={curriculum} />
+        <H3 style={{ marginTop: '1em' }}>{`Upcoming ${type} courses`}</H3>
```

Comment:
@ric9176 does "Upcoming React bootcamp courses" sounds good (bootcamp + courses)? It also breaks in two lines.  "Upcoming Advanced React courses" also breaks in two lines on desktop. "Next Advanced React courses" fits in one line